webpack/assets/javascripts/react_app/components/Bookmarks/components/BookmarkForm/index.js

```diff
@@ -1,15 +1,23 @@
 import { connect } from 'react-redux';
-
 import BookmarkForm from './BookmarkForm';
 import { submitForm } from '../../../../redux/actions/common/forms';
+import { selectAutocompleteSearchQuery } from '../../../AutoComplete/AutoCompleteSelectors';
+
+const mapStateToProps = (state, ownProps) => ({
+  initialValues: {
+    public: true,
+    query:
+      selectAutocompleteSearchQuery(
+        { autocomplete: { id: 'searchBar' }, ...state },
+        'searchBar',
+        ownProps
+      ) || '',
```

Comment:
tested and `selectAutocompleteSearchQuery(state, 'searchBar', { controller })` worked just fine

Reply:
@laviro Yes but it fails the bookmarks integration test and I can't figure out why :(

Reply:
@laviro Figured it out, had to add the `autocompleteReducer` in a couple places and then fix a React bug with Formik.  should be all set now :smile: